cmake/FindRust.cmake

```diff
@@ -248,6 +250,18 @@ else()
     )
 endif()
 
+if (_RUSTC_VERSION_RAW MATCHES "LLVM version: ([0-9]+)\\.([0-9]+)\\.([0-9]+)")
```

Comment:
do users need to care about the LLVM version.
I guess you added that because you have an usecase.

Reply:
My usecase is cross-language LTO. It requires that the LLVM version on both sides matches, so I need to know the version of LLVM used by rust.